lib/dor/services/client/user_version.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Dor
+  module Services
+    class Client
+      # API calls that are about user versions
+      class UserVersion < VersionedService
+        UserVersion = Struct.new(:version, :userVersion, keyword_init: true)
```

Comment:
I wonder if we should consider a different name here, since we have the same constant on lines 7 and 8.

Reply:
Changed.